tests/contrib/pymongo/test.py

```diff
@@ -406,6 +409,36 @@ def test_host_kwarg(self):
         assert client
 
 
+class TestPymongoPatchDisabledTracer(TestPymongoPatchDefault):
+    def get_tracer_and_client(self):
+        tracer = DummyTracer()
+        tracer.configure(enabled=False)
+        return tracer, self.get_client(tracer)
+
+    def test_delete(self):
+        pass
+
+    def test_insert_find(self):
+        # this test raises an Exception in pymongo/cursor.py
+        # self._datadog_trace_operation(operation) returns None when a tracer is disabled
+        super().test_insert_find()
```

Comment:
after the issue is resolved we need to basically duplicate `test_insert_find()` right? since the assertions need to be different (no spans created) ?

Reply:
Yup, this was a quick way of duplicating this error. I'll need to update `test_insert_find()`